dataactcore/local_secrets_example.yml

```diff
@@ -23,34 +23,24 @@ broker:
     use_aws: false
 
     # If using AWS, set your region here
-    aws_region: us-east-1
+    aws_region: aws-region
 
     # Add your AWS Key to use for sending SES emails
-    aws_access_key_id: ACCESSKEYID123
-    aws_secret_access_key: test123abc/123testsecretkeY
+    aws_access_key_id: keyId
+    aws_secret_access_key: accessKey
 
     # Name of AWS S3 bucket for uploaded broker files. Ignored if use_aws
     # is false. NOTE: the dummy value below MUST be changed to the correct
     # value if use_aws is true.
-    aws_bucket: sample-aws-bucket-name
-    
-    # S3 filenames for SF-133 file, only required if planning to load SF-133 table
-    sf_133_bucket: sample-sf133-bucket-name
-
-    # Region for pulling CFDA file, only required if planning to load CFDA file
-    cfda_region: us-east-1
-
-    # Bucket for pulling CFDA file, only required if planning to load CFDA file
-    cfda_bucket: sample-cfda-bucket-name
```

Comment:
Why were these removed? These are used

Reply:
Will add those back in

Reply:
realized this cfda_region and cfda_bucket was already in example_config.